less/variables.less

```diff
@@ -1,14 +1,17 @@
-
-// Color assignments 
+// Color assignments
 @font-color: @shark;
 @button-bg: @eastern-blue;
-@main-bg:  @white;
+@main-bg: @white;
 @navigation-border: @silver;
 @footer-bg: @sandy-beach;
 
 // Colors
-@white: #FFFFFF;
-@silver: #C0C0C0;
+@white: #ffffff;
+@silver: #c0c0c0;
 @shark: #212529;
-@eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@eastern-blue: #17a2b8;
+@sandy-beach: #ffebcd;
+
+// mobile design
+@mobile: (max-width: 500px);
+@tablet: (max-width: 800px);
```

Comment:
Great work on making custom colors and making both a mobile and tablet view, They both look great!